cvat-core/src/annotations-objects.js

```diff
@@ -261,6 +263,19 @@
             }, [this.clientID], frame);
         }
 
+        _saveAnnotationType(annotation_type) {
```

Comment:
It also important to update function ``updateTimestamp`` to this function considers annotationType